commission/reports/report_settlement_templates.xml

```diff
@@ -6,15 +6,15 @@
                 <t t-call="web.internal_layout">
                     <div class="page">
                         <div class="row mt32 mb32">
-                            <div class="col-xs-3" t-if="o.agent_id">
+                            <div class="col-3" t-if="o.agent_id">
```

Comment:
Why changing this?

Reply:
`col-xs-*` was replaced with `col-*` in bootstrap 4

Reply:
Ok